core/domain/collection_services.py

```diff
@@ -290,11 +292,14 @@ def get_learner_collection_dict_by_id(
                 summary.exploration_model_last_updated
             ) if summary else None,
             'thumbnail_icon_url': utils.get_thumbnail_icon_url_for_category(
-                summary.category),
+                summary.category) if summary else
```

Comment:
These two changes seem to be working around the problem, rather than fixing it. I haven't been able to reproduce this myself. Maybe we should talk about it some more. Can you revert these changes and list out steps in the conversation from starting a clean instance of Oppia to producing the bug so that I can also look into it and try to figure out with you what's going on?